R/transform_log.R

```diff
@@ -0,0 +1,20 @@
+#' transform_log
+#'
+#' log-transformation of a numeric vector. For details about log-transformation please see you basic school math textbook.
```

Comment:
Please remove the last sentence and either think of 

1. Why log transformations are typically used and/or
2. Hint to a Web link which describes the transformation in detail